stop_compaction_test.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2021 ScyllaDB
+import re
+
+from sdcm.cluster import BaseNode
+from sdcm.nemesis import StartStopMajorCompaction, StartStopScrubCompaction, StartStopCleanupCompaction, \
+    StartStopValidationCompaction, StartStopUpgradeCompaction, StartStopReshapeCompaction
+from sdcm.rest.storage_service_client import StorageServiceClient
+from sdcm.tester import ClusterTester
+from sdcm.utils.compaction_ops import CompactionOps
+
+
+class StopCompactionTest(ClusterTester):
```

Comment:
What is the purpose of the class ?
Does looks like pure functional test is that true ?

Reply:
Yes, these tests are functional (though the start/stop compaction methods can be used in longevity tests, I think). I tried to implement these test in dtest originally, but they were very flaky with the amounts of data used in dtest. We decided to move them to SCT for that reason.

Reply:
it looks good, only thing that bother me is that you will endup having tons of test id's and emails when running these test

Reply:
the worst thing is not the email but the tons of clusters that are going to be created. Per our discussion, we need to use subTest